modules/management-console-agent/src/main/java/org/apache/ignite/internal/agent/utils/AgentUtils.java

```diff
@@ -214,14 +215,56 @@ public static void authorizeIfNeeded(IgniteSecurity security, String name, Secur
      * Quietly closes given processor ignoring possible checked exception.
      *
      * @param proc Process.
+     * @param log Logger.
      */
-    public static void quiteStop(GridProcessor proc) {
+    public static void stopProcessor(GridProcessor proc, IgniteLogger log) {
         if (proc != null) {
             try {
                 proc.stop(true);
             }
-            catch (Exception ignored) {
-                // No-op.
+            catch (Exception ex) {
+                U.warn(log, ex.getMessage(), ex);
+            }
+        }
+    }
+
+    /**
+     * Creates agent processor.
+     *
+     * @param log Logger.
+     * @param clsName Processor class name.
+     * @param paramTypes The parameter array.
+     * @param args array of objects to be passed as arguments to the constructor call.
+     * @return Agent processor.
+     */
+    public static <T extends GridProcessor> T createProcessor(IgniteLogger log, String clsName, Class<?>[] paramTypes, Object... args) {
+        ClassLoader ldr = U.gridClassLoader();
+
+        try {
+            Class<?> mgrCls = ldr.loadClass(clsName);
+
+            return (T)mgrCls.getConstructor(paramTypes).newInstance(args);
+        }
+        catch (Exception ex) {
+            U.warn(log, "Failed to initialize GridGain Control Center agent processor: " + clsName, ex);
+        }
+
+        return null;
+    }
+
+    /**
+     * Quietly start given processor ignoring possible checked exception.
+     *
+     * @param proc Processor.
+     * @param log Logger.
+     */
+    public static void startProcessor(GridProcessor proc, IgniteLogger log) {
+        if (proc != null) {
+            try {
+                proc.start();
+            }
+            catch (Exception ex) {
+                U.warn(log, ex.getMessage(), ex);
```

Comment:
Impossibility to start a processor sounds more like an ERROR to me.

Reply:
Also some message would be nice.

Reply:
Fixed